### PR TITLE
Add additional information

### PR DESCRIPTION
**Pull request by defensivedepth**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,26 @@
 # Security Onion - Resources
 
-This repo contains the following resources:
+This repository contains the following resources:
 
-- Security Onion specific Sigma Rules
 
+## Security Onion specific Sigma Rules
+  - Location: `main` branch, sigma folder
+  - License: Elastic-2.0
```
Comment (dougburks) [inline README.md:8]: License: Elastic License 2.0 (ELv2) - https://securityonionsolutions.com/license

File: README.md [hunk 2]
```diff
@@ -1,6 +1,26 @@
...
+    
+This Sigma ruleset is maintained by Security Onion and is loaded by default into the Security Onion Detections' module.
```
Comment (dougburks) [inline README.md:10]: Do we need an apostrophe in `Detections' module`?

File: README.md [hunk 3]
```diff
@@ -1,6 +1,26 @@
...
+Generic event filters for process_creation, dns_query, file_create and more. Used by Security Onion to generate event filters for Elastic Defend events.
+
+Originally sourced from https://github.com/Neo23x0/sysmon-config and https://github.com/olafhartong/sysmon-modular
+
+
+## AI-Generated Detection Summaries
+  - Location: `generated-summaries-published` branch, detections-ai folder
+  - License: Elastic-2.0
+
+Summaries created by an LLM for Suricata, Sigma and YARA rules. Used by Security Onion in the Detections' module.
```
Comment (dougburks) [inline README.md:26]: Do we need an apostrophe in `Detections' module`?

File: README.md [hunk 4]
```diff
@@ -1,6 +1,26 @@
...
+  - License: MIT
+    
+Generic event filters for process_creation, dns_query, file_create and more. Used by Security Onion to generate event filters for Elastic Defend events.
+
+Originally sourced from https://github.com/Neo23x0/sysmon-config and https://github.com/olafhartong/sysmon-modular
+
+
+## AI-Generated Detection Summaries
+  - Location: `generated-summaries-published` branch, detections-ai folder
+  - License: Elastic-2.0
```
Comment (dougburks) [inline README.md:24]: License: Elastic License 2.0 (ELv2) - https://securityonionsolutions.com/license

**Review by dougburks — CHANGES_REQUESTED** [review]

**Review by TOoSmOotH — APPROVED** [review]

**Review by dougburks — APPROVED** [review]

**Pull request merged**